config/application.rb

```diff
@@ -118,6 +118,7 @@ class Application < Rails::Application
       Doorkeeper::Application.include ApplicationExtension
       Doorkeeper::AccessGrant.include AccessGrantExtension
       Doorkeeper::AccessToken.include AccessTokenExtension
+      Doorkeeper::OAuth::PreAuthorization.include OauthPreAuthorizationExtension
```

Comment:
I'm honestly not sure I've done this in the "correct" way, but it's temporary until https://github.com/doorkeeper-gem/doorkeeper/issues/1717 is resolved.

Reply:
Given that, I believe it is totally fine for now.